src/main/java/uk/gov/hmcts/befta/dse/ccd/definition/converter/EnvironmentURLUtils.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.befta.dse.ccd.definition.converter;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class EnvironmentURLUtils {
+
+    private static final String BASE_URL_PLACEHOLDER_REGEX = "\\$\\{(.*?)\\}";
```

Comment:
optional: one of my linting tools is reporting that the final escape sequence as: `Redundant character escape '\\}' in RegExp`.  i.e. the following will have the same result. 
```java
private static final String BASE_URL_PLACEHOLDER_REGEX = "\\$\\{(.*?)}";
```

Reply:
Resolved in https://github.com/hmcts/befta-fw/pull/123/commits/f7fbff8c44e377ce7f637f1419a516c10c184c61